packages/core/src/form-field/FormField.css

```
@@ -1,8 +1,8 @@
 .saltFormField {
   display: inline-grid;
   position: relative;
-  column-gap: var(--salt-size-adornmentGap);
-  row-gap: var(--salt-size-adornmentGap);
+  column-gap: var(--salt-spacing-100);
+  row-gap: var(--salt-spacing-100);
```

Couldn't we just do:
```suggestion
  gap: var(--salt-spacing-100);
```